--- drvrgsiftp.c.orig	2011-01-23 20:05:28.000000000 +0000
+++ drvrgsiftp.c	2011-01-22 17:18:01.000000000 +0000
@@ -2,6 +2,10 @@
 /*  This file, drvrgsiftp.c contains driver routines for gsiftp files. */
 /*  Andrea Barisani <lcars@si.inaf.it>                                 */
 /* Taffoni Giuliano <taffoni@oats.inaf.it>                             */
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
 #ifdef HAVE_NET_SERVICES
 #ifdef HAVE_GSIFTP
 
--- drvrnet.c.orig	2011-01-23 20:05:42.000000000 +0000
+++ drvrnet.c	2011-01-22 17:13:42.000000000 +0000
@@ -147,6 +147,9 @@
 
    
  */
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
 
 #ifdef HAVE_NET_SERVICES
 #include <string.h>
--- drvrsmem.c.orig	2011-01-23 20:05:57.000000000 +0000
+++ drvrsmem.c	2011-01-22 17:19:05.000000000 +0000
@@ -12,6 +12,10 @@
             no shmem segments remain, and last process terminates
 */
 
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
 #ifdef HAVE_SHMEM_SERVICES
 #include "fitsio2.h"                         /* drvrsmem.h is included by it */
 
--- cfileio.c.orig	2013-01-29 15:33:39.000000000 -0800
+++ cfileio.c	2013-01-29 15:34:28.000000000 -0800
@@ -10,6 +10,11 @@
 #include <ctype.h>
 #include <errno.h>
 #include <stddef.h>  /* apparently needed to define size_t */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
 #include "fitsio2.h"
 #include "group.h"
 
