--- drvrgsiftp.c.orig	2011-01-23 20:05:28.000000000 +0000
+++ drvrgsiftp.c	2011-01-22 17:18:01.000000000 +0000
@@ -2,6 +2,10 @@
 /*  This file, drvrgsiftp.c contains driver routines for gsiftp files. */
 /*  Andrea Barisani <lcars@si.inaf.it>                                 */
 /* Taffoni Giuliano <taffoni@oats.inaf.it>                             */
+#ifdef HAVE_CONFIG_H
+#include "fitsio_config.h"
+#endif
+
 #ifdef HAVE_NET_SERVICES
 #ifdef HAVE_GSIFTP
 
--- drvrnet.c.orig	2011-01-23 20:05:42.000000000 +0000
+++ drvrnet.c	2011-01-22 17:13:42.000000000 +0000
@@ -147,6 +147,9 @@
 
    
  */
+#ifdef HAVE_CONFIG_H
+#include "fitsio_config.h"
+#endif
 
 #ifdef HAVE_NET_SERVICES
 #include <string.h>
--- drvrsmem.c.orig	2011-01-23 20:05:57.000000000 +0000
+++ drvrsmem.c	2011-01-22 17:19:05.000000000 +0000
@@ -12,6 +12,10 @@
             no shmem segments remain, and last process terminates
 */
 
+#ifdef HAVE_CONFIG_H
+#include "fitsio_config.h"
+#endif
+
 #ifdef HAVE_SHMEM_SERVICES
 #include "fitsio2.h"                         /* drvrsmem.h is included by it */
 
--- drvrsmem.h.orig	2011-01-23 20:06:08.000000000 +0000
+++ drvrsmem.h	2011-01-22 17:10:37.000000000 +0000
@@ -18,6 +18,10 @@
 #include <sys/fcntl.h>
 #endif
 
+#ifdef HAVE_CONFIG_H
+#include "fitsio_config.h"
+#endif
+
 		/* configuration parameters */
 
 #define	SHARED_MAXSEG	(16)		/* maximum number of shared memory blocks */
--- fitsio2.h.orig	2011-01-23 20:06:23.000000000 +0000
+++ fitsio2.h	2011-01-22 17:08:40.000000000 +0000
@@ -1,6 +1,10 @@
 #ifndef _FITSIO2_H
 #define _FITSIO2_H
  
+#ifdef HAVE_CONFIG_H
+#include "fitsio_config.h"
+#endif
+
 #include "fitsio.h"
 
 /* 
