--- a/Makefile.in
+++ b/Makefile.in
@@ -116,7 +116,7 @@
 		${CC} $(LDFLAGS) $(DEFS) -o smem smem.o -L. -lcfitsio -lm -lz
 
 testprog:	testprog.o libcfitsio.a ${OBJECTS}
-		${CC} $(LDFLAGS) $(DEFS) -o testprog testprog.o -L. -lcfitsio -lm -lz ${LIBS}
+		${CC} $(LDFLAGS) $(DEFS) -o testprog testprog.o libcfitsio.a -lm -lz ${LIBS}
 
 fpack:		fpack.o fpackutil.o libcfitsio.a ${OBJECTS}
 		${CC} $(LDFLAGS) $(DEFS) -o fpack fpack.o fpackutil.o -L. -lcfitsio -lm -lz ${LIBS}
